config/jest/browser/open.mjs

```diff
@@ -7,6 +7,14 @@ import { setup } from 'jest-environment-puppeteer'
  * @param {import('jest').Config} jestConfig
  */
 export default async function browserOpen (jestConfig) {
+  const { maxWorkers } = jestConfig
+
+  // Increase maximum Node.js listeners for browserPerWorker
```

Comment:
I can sort of follow this but I think this would be a more useful comment if it covered what the number of listeners is increased to, and why?

Is this basically making sure we have a corresponding listener for each worker, with the number of workers coming from the Jest config? But then we need to add 1 for some reason – the main thread, or just it 'just in case'?

Reply:
Morning @36degrees can you take another look?

Hopefully explained it a bit better in the comment and commit